ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java

```diff
@@ -77,13 +77,17 @@ public Bundle build() {
 
   public static AdConfig adConfigWithNetworkExtras(Bundle networkExtras) {
     AdConfig adConfig = new AdConfig();
-    adConfig.setMuted(true); // start muted by default.
```

Comment:
@HuFangshuai wanted to confirm that if no network extras set to request, by not explicitly setting any muted flag are we leaving it to the creative setting ?
It seems to be different than when network extras is set but EXTRA_START_MUTED flag is not added, in which case. we are invoking setMute(false)

Reply:
fixed, declare unmute settings for ads.